README.md

```diff
@@ -39,24 +42,25 @@ AwsSsmEnv.load!(begins_with: "myapp.#{ENV['RACK_ENV']}.")
 
 ## Quick Start
 
-事前にAWS EC2 Parameter Storeにパラメータを登録しておく必要があります。
+It is necessary to register the parameters in `AWS EC2 Parameter Store` in advance.
 
 ```shell
-# 例) /myservice/staging/RDS_PASSWORDをSecureStringで登録
+# ex) register /myservice/staging/RDS_PASSWORD with SecureString
 aws ssm --region ap-northeast-1 put-parameter \
   --name /myservice/staging/RDS_PASSWORD \
   --type SecureString --value <secret value>
 ```
 
-AWSの認証情報を設定します。例えば、以下のように環境変数を利用したり、
+Set authentication information of AWS.   
```

Comment:
This is in sequential instructions. So I think we can put `Then,` here to show the sequence explicitly:

> Then, set authentication information of AWS.